src/main/java/Task.java

```diff
@@ -0,0 +1,26 @@
+public class Task {
+    /**
+     * Task class stores a String as name of the task and boolean indicating task completion
+     */
+
+    private String taskName;
+    private boolean done;
```

Comment:
As per coding standard, boolean variable name would be a name to point out that the variable is boolean. E.g ("isDone") would be good :D